packages/backend-function/src/function_env_type_generator.ts

```diff
@@ -84,6 +87,21 @@ export class FunctionEnvironmentTypeGenerator {
     this.writeShimFile(content);
   };
 
+  /**
+   * Clear existing files and subdirectories in the generated env directory
+   */
+  clearGeneratedEnvDirectory(): void {
```

Comment:
```suggestion
  clearGeneratedEnvDirectory = (): void => {
```

this is the preferred notation in the repo (see other functions around).
we do have a rule for this here https://github.com/aws-amplify/amplify-backend/blob/f6b1943b42d2e1669109f3411ff3c19bae14a2c0/.eslintrc.cjs#L76C6-L76C47 . I'm not sure why it didn't flag this.